src/scripts.js

```diff
@@ -81,6 +54,7 @@ function greetUser() {
 }
 
 function favoriteCard(event) {
+  const specificRecipe = "";
```

Comment:
I think this has to be a let? if we are going to reassign it below?

Reply:
actually i'm still working through this function, this variable is declared here and I haven't even used it yet - so yeah, work in progress